conf/examples/acm_hub_unreleased_image.yaml

```diff
@@ -1,3 +1,3 @@
 ---
 ENV_DATA:
-  acm_unreleased_image: "2.5.2-DOWNSTREAM-2022-07-29-15-56-49"
+  acm_unreleased_image: "2.7.0-DOWNSTREAM-2022-11-17-03-53-08"
```

Comment:
can you update image to `2.7.0-DOWNSTREAM-2023-01-07-13-58-25`

Reply:
done